packages/emotion/src/withStyle.tsx

```diff
@@ -153,11 +153,13 @@ const withStyle = decorator(
       )
     })
     hoistNonReactStatics(WithStyle, ComposedComponent)
-    // we have to pass these on, because sometimes we need to
+    // we have to pass these on, because sometimes users
     // access propTypes of the component in other components
-    // (mainly in the `omitProps` method)
+    // eslint-disable-next-line react/forbid-foreign-prop-types
     WithStyle.propTypes = ComposedComponent.propTypes
     WithStyle.defaultProps = ComposedComponent.defaultProps
+    // @ts-expect-error These static fields exist on InstUI components
+    WithStyle.allowedProps = ComposedComponent.allowedProps
```

Comment:
again, before this fix I think `@withStyle` was erasing allowedProps

Reply:
Nice catch!